pkg/util/containers/collectors/kubelet.go

```diff
@@ -32,7 +32,7 @@ func (c *KubeletCollector) Detect() error {
 		return errors.New("Kubernetes feature is deactivated")
 	}
 
-	util, err := kubelet.GetKubeUtil()
+	util, err := kubelet.GetKubeUtilWithRetrier()
```

Comment:
`GetKubeUtilWithRetrier()` doesn't return an error:
`func GetKubeUtilWithRetrier() (KubeUtilInterface, *retry.Retrier)`

Reply:
[This should be better](https://github.com/DataDog/datadog-agent/commit/e06a22e61157ba1781ee37cb12c90c00a5d8b656#diff-6115d750f88e369e6d97d6f53b98bbc95ffb62f2e29494ea0202f30367085852).